chapel.html.markdown

```diff
@@ -13,42 +13,60 @@ as well as multi-kilocore supercomputers.
 
 More information and support can be found at the bottom of this document.
 
+You can refer to the official site for [latest version](https://chapel-lang.org/docs/master/primers/learnChapelInYMinutes.html) of this document.
+
 ```chapel
+/*
```

Comment:
Is this a leftover from chapel/tests? Seems odd here.
Should probably go at the bottom, maybe in its own section.

Reply:
No, it's from the official page. I just add the timestamp at the last line just in case this doc gets outdated and confuses people ( this happened before).